content/guest-essays/uncertainty-and-utilitarianism.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Uncertainty and Utilitarianism"
+book: "Uncertainty and Utilitarianism"
+year: "2022"
+date: 2022-08-08T10:10:55-04:00
+author: "Krister Bykvist"
+draft: false
+layout: "book"
+menu: "guest"
+weight: 8
+image: "/img/essays/criminal-law.jpg"
+imageCaption: "Uncertainty and Utilitarianism"
+summary: "summary"
```

Comment:
@rychappell - please add one in sometime, or email me and I'll push the changes 🤝

Reply:
done!